docs/docs/external/getting-started/test-network/ledger-initialization.md

```diff
@@ -181,10 +181,34 @@ Bootstrap the Corda network and application states as follows:
     ./scripts/initAsset.sh
     ```
 
-### Next Steps
+### Initializing the Besu Networks
+
+Let's assume that `network1` can either manage NFT `AliceERC721` or Hybrid `AliceERC1155` tokens, while `network2` manages fungible `BobERC20` tokens. Here we use account `1` for Alice and account `2` for Bob in both neworks. To prepare Besu networks for asset exchange, navigate to the `samples/besu/besu-cli` and then follow the steps in next subsections.
+
+#### Configuring the Besu-CLI
+
+Create a `config.json` file by copying the `config.template.json`, keep the default values for managing `AliceERC721` tokens in `network1` and `BobERC20` tokens in `network2`.
+If you want to change the token type used in the `network1` to Hybrid `AliceERC1155` tokens, in `config.json` update `tokenContract` field with value `"../simpleasset/build/contracts/AliceERC1155.json".`
+  
+#### Bootstrapping Network and Application State
+
+Finally, to prepare both `network1` and `network2` for interoperation, run:
 
-The test networks are now configured and their ledgers are initialized. You can now run the [asset exchange flows](../interop/asset-exchange.md).
+* for default configuration, run:
```

Comment:
"If you wish to test the default exchange (ERC-20 tokens for ERC-721 tokens), run:"